.github/workflows/release.yml

```diff
@@ -31,12 +31,12 @@ jobs:
           pip install yq
 
           cd /usr/bin
-          curl -s "https://github.com/kubernetes-sigs/kustomize/blob/kustomize/v4.0.4/hack/install_kustomize.sh 4.0.4"  | bash
+          curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/kustomize/v4.0.4/hack/install_kustomize.sh"  | bash
           cd ${GITHUB_WORKSPACE}
       - name: Build image, generate deployments and push git tag
         run: |
           git config --global user.name "Mykhailo Kuznietsov"
           git config --global user.email "mkuznets@redhat.com"
 
           export GITHUB_TOKEN=${{ secrets.CHE_INCUBATOR_BOT_GITHUB_TOKEN }}
-          ./make-release.sh --version ${{ github.event.inputs.version}}
+          bash -x ./make-release.sh --version ${{ github.event.inputs.version}}
```

Comment:
is -x here just for debugging or better to always has it?

Reply:
tbf I can remove it from here for now